sigs.yaml

```diff
@@ -1076,8 +1076,8 @@ sigs:
     label: docs
     leadership:
       chairs:
-      - name: Andrew Chen
-        github: chenopis
+      - name: Jared Bhatti  
```

Comment:
Consider adding Andrew as an Emeritus lead? :)


Example - https://github.com/kubernetes/community/blob/d335e174085078db1a3b12ae86753c66274d1813/sigs.yaml#L325

Reply:
@nikhita What a wonderful idea! 💛

@Bradamant3 I updated your current employer as well, if that's OK with you. Please let me know.

Reply:
lol just saw this after I started what I thought would be a follow-up PR to do just this. Thank you! `<subsides>`

Reply:
I guess it should be changed here as well? :grimacing: 

https://github.com/kubernetes/community/blob/20c1e33a741f238ec0428f7fdc096ca1ff0d862d/sigs.yaml#L2644-L2646